private.tf

```diff
@@ -1,31 +1,37 @@
+module "private_subnet_label" {
+  source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.2.1"
+  namespace = "${var.namespace}"
+  stage     = "${var.stage}"
+  name      = "private"
```

Comment:
I think this should technically be `${var.name}`, and `private` is an `attribute`.

Reply:
Why do we need an additional label? e.g. there's `private_label`

Reply:
We ageed this naming convention recently. That `subnets` should have `public` and `private` names.
https://app.asana.com/0/386406220258003/429148314577190

Reply:
agreed. They should. But isn't there a problem if we don't honor the caller's `name` at all? We're just discarding it here. I agree we discussed this... but I don't know if I fully grasped the implications.

Reply:
Then we can return to using `private_label`.

Reply:
Are we going to reuse the same subnets for different applications in the same namespace/stage?

If yes, then `name` should be removed from the Name of subnets as we have now.
If not, we need to honor the caller provided `name`.

So we could either have `ap-infra-public-us-east-1e` or `ap-infra-jenkins-public-us-east-1e`.

We are using `name` in everything else we create for the subnets - Routing Table `ap-infra-jenkins-public` and Network ACL `ap-infra-jenkins-public`.

@osterman @comeanother  ??